src/bench/data.cpp

```diff
@@ -8,7 +8,7 @@ namespace benchmark {
 namespace data {
 
 #include <bench/data/block413567.raw.h>
-const std::vector<uint8_t> block413567{block413567_raw, block413567_raw + sizeof(block413567_raw) / sizeof(block413567_raw[0])};
+const std::vector<uint8_t> block413567{block413567_raw, block413567_raw + std::size(block413567_raw)};
```

Comment:
```suggestion
const std::vector<uint8_t> block413567{std::begin(block413567_raw), std::end(block413567_raw)};
```

seem overly verbose to do pointer arithmetic here. Same for all other occurrences in this pull

Reply:
Done, added an extra commit for this, again _before_ the `std::size` substitution to minimize retouching of lines.